content/en/docs/concepts/overview/object-management-kubectl/kustomization.md

```diff
@@ -0,0 +1,752 @@
+---
+title: Declarative Management of Kubernetes Objects Using Kustomization
+content_template: templates/concept
```

Comment:
Declarative Management of Kubernetes Objects Using Kustomize

Reply:
updated